ui/app/styles/core/forms.scss

```diff
@@ -13,7 +13,6 @@ label {
 
 .masked-font {
   font-family: obscure, text-security-square;
-  font-size: $size-medium;
```

Comment:
removed because it was making the masked-input larger than the key column

Reply:
thank you for fixing this!